docs/sql-migration-guide-hive-compatibility.md

```diff
@@ -51,6 +51,22 @@ Spark SQL supports the vast majority of Hive features, such as:
 * Explain
 * Partitioned tables including dynamic partition insertion
 * View
+  * If column aliases are not specified in view definition queries, both Spark and Hive will
+    generate alias names, but in different ways. In order for Spark to be able to read views created
+    by Hive, users should explicitly specify column aliases in view definition queries. As an
+    example, Spark cannot read `v1` created as below by Hive.
+
+    ```
+    CREATE TABLE t1 (c1 INT, c2 STRING);
+    CREATE VIEW v1 AS SELECT * FROM (SELECT c1 + 1, upper(c2) FROM t1) t2;
```

Comment:
Could you simplify more by removing `CREATE TABLE` and using the following view creation?
```sql
CREATE VIEW v1 AS SELECT * FROM (SELECT c + 1, upper(c) FROM (SELECT 1 c) t1) t2;
```

Reply:
@dongjoon-hyun i was thinking, calling upper on a int column is probably not very intuitive :-)
What do you think about adding a string literal in the projection ?

```
SELECT c + 1, upper(d) FROM select 1 c, 'test' as d 
```